modules/bullet/bullet_physics_server.h

```diff
@@ -51,44 +51,61 @@ class BulletPhysicsServer3D : public PhysicsServer3D {
 	friend class BulletPhysicsDirectSpaceState;
 
 	bool active = true;
-	char active_spaces_count = 0;
-	Vector<SpaceBullet *> active_spaces;
+	bool using_threads;
+	bool doing_sync;
 
-	mutable RID_PtrOwner<SpaceBullet> space_owner;
-	mutable RID_PtrOwner<ShapeBullet> shape_owner;
-	mutable RID_PtrOwner<AreaBullet> area_owner;
-	mutable RID_PtrOwner<RigidBodyBullet> rigid_body_owner;
-	mutable RID_PtrOwner<SoftBodyBullet> soft_body_owner;
-	mutable RID_PtrOwner<JointBullet> joint_owner;
+	Set<SpaceBullet *> active_spaces;
```

Comment:
to make it uniform. because we also do this in the Default Physics.

Reply:
I don't see a reason why this should be a `Set`, so let's keep it the way it was. It's possible that Godot Physics needs a Set for a specific reason, otherwise it can be changed to a `Vector` too (but not in this PR).

Reply:
This was marked as resolved, but wasn't addressed.